hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/S3BucketManagerImpl.java

```diff
@@ -166,6 +170,10 @@ public boolean createOzoneVolumeIfNeeded(String userName)
               .setVolume(ozoneVolumeName)
               .setQuotaInBytes(OzoneConsts.MAX_QUOTA_IN_BYTES)
               .build();
+      if (isRatisEnabled) {
```

Comment:
Why do we need to call applyCreateVolume separately? Is this being pushed through Ratis consensus correctly?

Reply:
S3Bucket create internally calls createVolume. As now we have not separated s3 createbucket into 2 phase, we need to do this (In applyTransaction createBucket is called now). As createVolume when ratis is enabled it does not apply to OM DB. So, we need to call apply also here.